libs/bower/README.md

```diff
@@ -0,0 +1,21 @@
+# How to add dependencies with bower in ILIAS
```

Comment:
These changes can be ignored in this PR since the dependency-management using bower will be part of another PR